front/app/components/admin/UserBlockModals/messages.ts

```diff
@@ -0,0 +1,66 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  header: {
+    id: 'app.containers.Admin.Users.userBlockModal.header',
+    defaultMessage: 'Block user',
+  },
+  subtitle: {
+    id: 'app.containers.Admin.Users.userBlockModal.subtitle',
+    defaultMessage:
+      'The selected user won’t be able to login to the platform for {ninetyDays}. If you wish to revert this, you can unblock them from the list of blocked users.',
+  },
+  ninetyDays: {
+    id: 'app.containers.Admin.Users.userBlockModal.ninetyDays',
+    defaultMessage: '90 days',
+  },
+  reasonLabel: {
+    id: 'app.containers.Admin.Users.userBlockModal.reasonLabel',
+    defaultMessage: 'Reason',
+  },
+  reasonLabelTooltip: {
+    id: 'app.containers.Admin.Users.userBlockModal.reasonLabelTooltip',
+    defaultMessage: 'This will be comunicated to the blocked user.',
+  },
+  blockInfo: {
+    id: 'app.containers.Admin.Users.userBlockModal.blockInfo',
+    defaultMessage:
+      'The content of this user won’t be removed through this action. Don’t forget to moderate this user’s content if needed.',
```

Comment:
The content of this user won’t be removed through this action. Don’t forget to moderate this user’s content if needed.
```suggestion
      'The content of this user won’t be removed through this action. Don’t forget to moderate their content if needed.',
```

To reduce repetition.